common/src/main/java/org/opensearch/ml/common/transport/controller/MLUndeployModelControllerAction.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.common.transport.controller;
+
+import org.opensearch.action.ActionType;
+
+public class MLUndeployModelControllerAction extends ActionType<MLUndeployModelControllerNodesResponse> {
```

Comment:
So user can undeploy model controller separately ?  Which mean the model deployed , but undeploy the model controller, then no rate limiting any more?

Reply:
The deploy and undeploy of the model controller are passive behavior. User cannot actively trigger them.

Reply:
Add some comments to explain when will "deploy and undeploy of the model controller" be triggered ?

Reply:
Ack